.changeset/nervous-timers-hide.md

```diff
@@ -0,0 +1,208 @@
+---
+'@leafygreen-ui/select': minor
+---
+
+- Exports `getLGSelectTestUtils`, a util to reliably interact with `LG Select` in a product test suite. [LG-4129](https://jira.mongodb.org/browse/LG-4129)
+
+## getLGSelectTestUtils()
```

Comment:
TBH this is a bit overwhelming for release notes. We should summarize it here and link to the README

Reply:
ok!